crates/virtio-queue/src/state.rs

```diff
@@ -242,6 +242,10 @@ impl QueueStateT for QueueState {
         self.max_size
     }
 
+    fn actual_size(&self) -> u16 {
+        std::cmp::min(self.max_size, self.size)
```

Comment:
Based on some recent changes, the `size` of the queue can not be greater than `max_size` anymore, it is documented [here](https://github.com/rust-vmm/vm-virtio/blob/main/crates/virtio-queue/src/queue.rs#L127). [Here](https://github.com/rust-vmm/vm-virtio/pull/112/commits/3c84b4cca454de1b3ee2d77bfe02cdd60ad6c644) is the commit where the `actual_size` was removed. If we want to have a method that returns the size, can we rename it to `size()` and return `self.size`?

Reply:
Sure, will rename it to `size()`.